crates/core_arch/src/x86/avx512f.rs

```diff
@@ -10524,7 +10524,7 @@ pub unsafe fn _mm512_kxor(a: __mmask16, b: __mmask16) -> __mmask16 {
 /// [Intel's documentation](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=knot_mask16&expand=3233)
 #[inline]
 #[target_feature(enable = "avx512f")]
-#[cfg_attr(test, assert_instr(not))] // generate normal not code instead of knotw
+//#[cfg_attr(test, assert_instr(xor))] // generate normal not code instead of knotw
```

Comment:
Please don't leave commented-out code. Either keep the line with `xor` or delete it entirely.